### ISSUE TITLE
How to change x-ui database directory in source codes?

### ISSUE THREAD

**Issue by flightlover** [alireza0/x-ui#124]
Hello
I want to change the directory of databese which x-ui is using with. The current directory is `/etc/x-ui/x-ui.db` . Where can I change it in source code? I don't find it in source codes.
Thanks

**Comment by alireza0** [comment]
https://github.com/alireza0/x-ui/blob/main/config/config.go#L49

**Comment by flightlover** [comment]
Thanks
I edited it to different directories several times but nothing was changed in practice. e.g. I changed that block of code to:
```
func GetDBPath() string {
	return fmt.Sprintf("/etc/x-ui2/x-ui2.db")
}
```
then I reinstalled x-ui, but the database directory is still `/etc/x-ui/x-ui.db` and it didn't change to `/etc/x-ui2/x-ui2.db`. Also in another case the block of code was changed to:
```
func GetDBPath() string {
        return fmt.Sprintf("/etc/xyz/%s/%s.db", GetName(), GetName())
}
```
but the database directory is still `/etc/x-ui/x-ui.db` after reinstalling and it didn't change to `/etc/xyz/x-ui/x-ui.db`.

Finally if I want to change the directory to `/etc/x-ui2/x-ui2.db`, what exactly should I do?
Thanks.
